lib/team_data_filterer.rb

```diff
@@ -3,30 +3,6 @@
 
 module TeamDataFilterer
```

Comment:
I was trying to minimize refactoring to keep the pull request small...mind if I take care of that as a follow-up?